resources/charts/dataspace-connector/values.yaml

```diff
@@ -27,25 +27,17 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+ports: {}
```

Comment:
It would be cool to have here an example how the configuration could be look like

Reply:
Maybe same for volumes & volumeMounts

Reply:
@bscholtes1A You did it for volumes and VolumeMounts. Did you explicitly omit the ports or was that a mistake?

Reply:
> @bscholtes1A You did it for volumes and VolumeMounts. Did you explicitly omit the ports or was that a mistake?

My bad, I missed this one. 
Corrected!